src/components/inputs/TextInput.js

```diff
@@ -209,6 +225,25 @@ export default class TextInput extends BaseInput {
     }
   }
 
+  renderTitle() {
+    const {floatingPlaceholder, title, titleColor} = this.props;
+
+    if (!floatingPlaceholder && title) {
+      const marginBottom = Constants.isIOS ? 5 : 4;
+      const titleFontSize = 13;
+      const capitalizedTitle = this.capitalize(title);
+
+      return (
+        <Text
+          style={{top: 0, color: titleColor, marginBottom}}
```

Comment:
inline style is less explanatory, we should use it just for changing styling (titleColor), the rest should 

instead of defining style variables, create a style object using StyleSheet.
it's cleaner and better for performance.

Reply:
already changed